ConsentViewController/Classes/SourcePointClient/GDPRPrivacyManagerViewResponse.swift

```diff
@@ -26,33 +26,33 @@ struct GDPRVendor: Decodable {
     let consentCategories, legIntCategories: [Category]
     let iabSpecialPurposes, iabFeatures, iabSpecialFeatures: [String]
     
-    enum Keys: String, CodingKey {
-        case policyUrl, vendorId, name
-        case iabId
-        case iabSpecialPurposes, iabFeatures, iabSpecialFeatures
-        case description, cookieHeader
-        case vendorType
-        case consentCategories, legIntCategories
-    }
-    
     init(from decoder: Decoder) throws {
         let container = try decoder.container(keyedBy: Keys.self)
-        do {
-            policyUrl = try container.decodeIfPresent(URL.self, forKey: .policyUrl)
-        } catch {
+        name = try container.decode(String.self, forKey: .name)
+        if let tempPolicyStr = try container.decodeIfPresent(String.self, forKey: .policyUrl) {
```

Comment:
@Nevazhnovu you could use the `try?` operator here too and avoid the `if let else`